### PR TITLE
testing/lxd: upgrade to 2.20

### PR DESCRIPTION
**Pull request by vifino**

### CODE REVIEW

File: testing/lxd/APKBUILD [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
+# Contributor: Adrian Pistol <vifino@tty.sh>
```
Comment (rgdoliveira) [inline testing/lxd/APKBUILD]: I think the "Contributor" field is for who contributed the package to Alpine

Comment (vifino) [reply testing/lxd/APKBUILD]: Not for those who contributed to the package?

Comment (jirutka) [reply testing/lxd/APKBUILD]: Yes, who created the abuild or significantly improved. Package bump doesn't fit this category.

Comment (vifino) [reply testing/lxd/APKBUILD]: Ah. I did contribute more than just this to the LXD abuild, fixing it up, but I'll recommit without that line.

Comment (jirutka) [reply testing/lxd/APKBUILD]: However, the definition is currently very fuzzy and it's actually not used anywhere, so we're thinking about redefining it.

**Comment by vifino** [comment]
Alright. Change made.

**Pull request closed without merging**